gitignore

```diff
@@ -1,21 +1,49 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-if [ -z "${1:-}" ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
-    echo -e "Generate .gitignore file for the language you prefer.\nUses https://gitignore.io\n"
+# colors
+NC='\033[0m' # No Color
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+
+
+if [ -z "${1:-}" ] || [[ ${@} =~ "," ]] || printf '%s\n' ${@} | grep -q -P '^--?h(elp)?$'
+then
+    echo "Generate .gitignore file for the language you prefer."
+    echo -e "Uses https://gitignore.io\n"
     echo "Usage:"
-    echo -e "  gitignore <LANGUAGE>\n"
+    echo -e "  gitignore <environments> \n"
     echo "For example:"
-    echo "  gitignore python"
+    echo "  gitignore macos python pycharm"
     exit 1
 fi
 
-response="$(curl -sL https://www.gitignore.io/api/${1})"
+# implode all arguments by using ',' as delimiter and remove trailing ','
+templates=$(printf '%s,' "${@}")
+response="$(curl -sL https://www.gitignore.io/api/${templates%,})"
 
-if [[ $response =~ "#!! ERROR" ]]; then
-    echo -e "Could not create .gitignore. Please make sure you entered a valid language.\nYou can view all the valid languages at:\nhttps://www.gitignore.io/api/list?format=lines"
+if [[ $response =~ "#!! ERROR" ]]
+then
+    echo -e "$RED Could not create .gitignore. Please make sure you entered a valid language.\nYou can view all the valid languages at:\nhttps://www.gitignore.io/api/list?format=lines $NC"
```

Comment:
let's also change the error message now that we are supporting multiple environments and update the link that gives us valid languages to one that gives us valid environments.

Reply:
The link is for retrieving the valid environments is same as previous. Please suggest error message if you got any or the below would do just fine ?

> Could not create .gitignore. Please make sure you entered a valid environment.\nYou can view all the valid environments at:\nhttps://www.gitignore.io/api/list?format=lines

Reply:
yeah this one is fine